docs/integrations/files/parquet.mdx

```diff
@@ -0,0 +1,39 @@
+---
+title: Upload Parquet files to MindsDB
+sidebarTitle: Parquet
+---
+
+You can upload Parquet files of up to 500MB in size to MindsDB.
```

Comment:
Should we add a sentence and mention that once the file is uploaded it looks like a table inside the files database?

Reply:
Ok. Added this:
```
Parquet files are stored in the form of a table inside MindsDB.
```